### ISSUE TITLE
RStudio not loading/running slow/hanging code

### ISSUE THREAD

**Issue by avierse** [rstudio/rstudio#10892]
After weeks of RStudio working fine on my computer, I opened it this morning and found that code I entered would "hang" in the console and not execute. After uninstalling and re-installing numerous times, code would eventually execute but it would take 15 minutes+ for simple commands such as x <- 4 to run. After more uninstalling and re-installing and restarting my computer, currently, RStudio will not even open. It is stuck on the "taking longer to start than usual" screen, and has been for the past 10 minutes. 

I changed nothing about R or my computer since I last logged in on Friday, when things were working smoothly. I'm not sure what else to try.

**Comment by kevinushey** [comment]
Have you already tried resetting RStudio's state? https://support.rstudio.com/hc/en-us/articles/200534577-Resetting-RStudio-Desktop-s-State

**Comment by avierse** [comment]
I tried renaming the folder RStudio is stored in...but nothing improves. I may be missing a step though, I do not fully understand the directions you linked. What do I do after I rename the folder?

**Comment by ronblum** [comment]
@avierse That should be it—when RStudio is restarted, it will created the necessary directories. Note that a full reset might require removing several directories; for example, on Windows you might need to rename both `%localappdata%\RStudio` and `%appdata%\RStudio`.

What environment are you running in? Specfiically:

### System details

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       :

**Comment by avierse** [comment]
Hi Ron,
Thanks for your response. I'm running the below:

**Comment by avierse** [comment]
And just to clarify -- when you say "rename both %localappdata%\RStudio and %appdata%\RStudio", does that just entail manually renaming the RStudio folder, or is there another step to it?

Thanks!

**Comment by avierse** [comment]
Yes -- when I am using my local (C: ) drive and not opening any files through RStudio in the organization's R-drive and when my working directory is not set to a space in the organization's R-drive, all works smoothly. When I do either of these things (open files or set my wd to somewhere in the R-drive), that's when it bugs out.

**Comment by ronblum** [comment]
For renaming the directories: Yep, that's all! It's just to back up the original directory.

For the network drive: What happens if you use the network path, e.g. `\\remote-drive\Users\RonBl`? Usually when we see issues it's in this situation and they some resolve when using the mapped drive, but at least we'd have another data point.

**Comment by avierse** [comment]
Thanks!
@ronblum 

I renamed both Rstudio folders in my local and roaming app data -- still the same issue. RStudio works until I open a project from the organization's Rdrive.

I was previously using the most recent version. 

And it seems to work on all drives that I've tested, except for the Rdrive.

**Comment by avierse** [comment]
Update: It's working now? I renamed the local folders, then Rstudio bugged out again, so I deleted the non-renamed Rstudio local folders and restarted the session and now it's working (so far)?

**Comment by ronblum** [comment]
@avierse That's good to hear, that at least it's now working! Let us know if the issue returns. I'll leave this open for the moment, in case bugs out again in the near term.

**Comment by avierse** [comment]
@ronblum thanks so much! I really appreciate the idea support here, it's been a relief in what otherwise was a very frustrating situation!

**Comment by ronblum** [comment]
@avierse Glad to help!

**Comment by matguidi** [comment]
I have the same issues on a Linux (Manjaro) machine. I deleted all the configuration folders, tried to reinstall, but to no avail so far. I don't know if that is related, but I first encountered this issue after having installed Quarto, just for the sake of trying it. I wasn't able to run Quarto code, and from that moment on I've not been able to run any R code either.

**Comment by ronblum** [comment]
@matguidi We don't support Manjaro, but what happens if you do the following:
1) Disable Quarto: <kbd>Global Options -> R Markdown -> Quarto</kbd>, then uncheck `Enable Quarto preview`
2) If that doesn't work, what happens if you remove Quarto?

**Comment by ronblum** [comment]
@matguidi You're welcome—I'm glad the reinstall worked!

**Comment by avierse** [comment]
Glad this thread can be of help! Interestingly, although my issue has been resolved for accessing RStudio files in my organization's R Drive via my in-office Desktop and also by remoting into my in-office Desktop, I am still encountering the "hanging" issue when I try and access files via my laptop's mapped R Drive. I've been uninstalling/reinstalling and resetting RStudio's state, but it's not working for this approach. 

Worse comes to worse, I can still access RStudio via a remote desktop, but it'd be nice to also get to the bottom of this mapped R-drive issue. Any additional ideas for this one? Thanks!

**Comment by ronblum** [comment]
@kevinushey or @mikebessuille Do you have any suggestions for followups, such as any more information or things to try out?
(Asking the two of you because you've been involved in previous issues under `remote storage`.)

**Comment by kevinushey** [comment]
It might also be worth checking whether you encounter the same issue with older releases of RStudio.

https://www.rstudio.com/products/rstudio/older-versions/

**Comment by avierse** [comment]
Thanks, @kevinushey . I tried version 2022.02.0 as well as version 2021.09.2 and unfortunately the issue remains the same.

**Comment by avierse** [comment]
@kevinushey @ronblum any other thoughts on how to troubleshoot this? The issue persists, although I'll have a few hours here and there where everything works fine before crashing again. I've narrowed the issue down to R-drive access (I'll encounter the same problem across devices, so it's not a hard drive issue) and alerted my office tech team, but have at this time run out of fixes to try. 

Thanks in advance for any support!
